inference/enterprise/workflows/entities/steps.py

```diff
@@ -676,15 +855,26 @@ def validate_field_binding(self, field_name: str, value: Any) -> None:
 
 
 class BarcodeDetection(BaseModel, StepInterface):
```

Comment:
Should we rename this BarcodeModel since it inherits from BaseModel? (At the least should be BarcodeDetector vs Detection to match the part of speech of other blocks)

Reply:
that's `pydantic.BaseModel`.
`BarcodeDetector` is fine, we would also need to change `QRCodeDetection` in the same way

Reply:
ok,  changed. 
Also made the change non-breaking by double-aliasing step type - we will sun-set old ones in some time

Reply:
Also wonder if eg YoloWorld should be called YoloWorldModel